packages/tofu/src/util.js

```diff
@@ -42,7 +42,9 @@ function getNameFromNode (node) {
     return node.name
   } else if (node.type === 'ThisExpression') {
     return 'this'
-  } else {
+  } else if (node.type === 'PrivateName') {
+    return `#${node.id.name}`
+  }else {
```

Comment:
ESLint doesn't enforce this.  If we want to worry about it, we should adopt `prettier`.

Reply:
I'd advocate for them being authoritative--with the goal being eliminating all mention of style issues in PRs.

Reply:
I agree with both of you :sweat_smile: 
- machines should do the formatting
- we should do the right thing even if it's possible not to
- some level of automation is great. It stops being great when it gets in the way (eg. prettier demanding some variable names are not in use)
- style comments in PRs are a waste of time
- the reviewer's time is wasted but there's not much the reviewer can do to prevent it

I say we use autoformatting locally even if not enforced. It's what I've been doing and somehow it picked up on the existing config and formatted consistently with the existing code.
Would be great to not have to rely on being lucky like that tho. :) 

Let's close this conversation regardless of outcomes for the PR. I see a branch introducing prettier. Any style conversations can continue there.